apps/explorer/src/ui/CoinBalance.tsx

```diff
@@ -4,24 +4,25 @@
 import { useFormatCoin, CoinFormat } from '~/hooks/useFormatCoin';
 import { Amount, type AmountProps } from '~/ui/Amount';
 
-export interface CoinBalanceProps extends AmountProps {}
+export interface CoinBalanceProps extends Omit<AmountProps, 'symbol'> {
+    coinType?: string | null;
```

Comment:
Intentionally differentiating `coinType` and `symbol`, as they mean different things and aren't interchangeable.